R/ResamplingSpCVEnv.R

```diff
@@ -0,0 +1,121 @@
+#' @title Environmental Block Cross Validation Resampling
+#'
+#' @format [R6::R6Class] inheriting from [Resampling].
+#' @import mlr3
+#'
+#' @description
+#' Environmental Block Cross Validation.
```

Comment:
We should add a bit more info here and in `@details`.

Reply:
Specifically to outline the difference compared to `spcv-kmeans`.

We might also rename `spcv-kmeans` since we now have two approaches using kmeans.

`spcv-coords`?

Reply:
I will update the documentation. `spcv-coords` sounds good.